.pre-commit-config.yaml

```diff
@@ -3,6 +3,7 @@ repos:
     rev: v5.0.0
     hooks:
       - id: trailing-whitespace
+        exclude: '^.*?/__snapshots__/.*\.ambr$'
```

Comment:
I propose to exclude these files globally, like here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/.pre-commit-config.yaml#L50

Reply:
https://github.com/wemake-services/wemake-python-styleguide/pull/3421 let me know what you think.

Reply:
ah, did I musunderstand and you want me to move this glob to the global section, not change the style guide?

Reply:
4231391